server/scripting/stubs/server.lua

```diff
@@ -0,0 +1,649 @@
+--  old one running.
+
+-- Usage references:
+-- https://longturn.readthedocs.io/en/latest/Contributing/style-guide.html
+-- https://luals.github.io/wiki/definition-files
+-- https://luals.github.io/wiki/annotations/#documenting-types
+-- https://taminomara.github.io/sphinx-lua-ls/index.html#autodoc-directives
+-- https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#rst-primer
+
+--- The server module is for interacting with the server, and getting
+--- configuration details and settings.
+---
+--- !doctype table
+--- @class server
+server = {}
+
+--- Instructs the server to save the game. As with the
+--- :ref:`/save <server-command-save>` command, if filename is empty, the
+--- server chooses an appropriate filename.
+---
+--- @param filename string The filename to use, or nil to use a default.
+function server.save(filename) end
+
+--- 
+--- @return boolean started True, if the game has been started.
+function server.started() end
+
+--- Same as :lua:func:`Player:civilization_score`
+--- @param player Player
+--- @return int score
+function server.civilization_score(player) end
+
+--- Request player's client to play certain track.
+--- @param player Player The player who should play the track.
+--- @param tag string The tag of the track to play.
+--- @return boolean played True, if it played.
+function server.play_music(player, tag) end
+
+--- 
+--- @param String name The name of the :ref:`server setting <server-options>`.
+--- @return String value The value of a server setting as a string. 
+function server.setting.get(name) end
+
+
+--- Send event notifications to players. 
+---
+--- See the :ref:`E module <script-api-events>` for a list of possible event
+--- types.
+---
+--- !doctype table
+--- @class notify
+notify = {}
+
+--- Send all players a message with type :ref:`E.SCRIPT <script-api-events>`.
+---
+--- @param message String A :lua:func:`string.format`-style message.
+--- @param ... any A set of values to insert into the string template.
+function notify.all(message, ...) end
+
+--- Send a player a message with type :ref:`E.SCRIPT <script-api-events>`.
+---
+--- @param player Player The player to send the event message to.
+--- @param message String A :lua:func:`string.format`-style message.
+--- @param ... any A set of values to insert into the string template.
+function notify.player(player, message, ...) end
+
+--- Send a specific player or all players (if player is nil) a message with
+--- an arbitrary type, optionally drawing attention to a particular tile.
+---
+--- @param player Player The player to send the event message to, or nil for all players.
+--- @param tile Tile The tile to highlight, or nil for no tile.
+--- @param event Event The :ref:`Event Category <script-api-events>`.
+--- @param message String A :lua:func:`string.format`-style message.
+--- @param ... any A set of values to insert into the string template.
+function notify.event(player, tile, event, message, ...) end
```

Comment:
I wonder why this renders wierd vs player(). Looks like embassies and research do the same thing.
<img width="752" height="553" alt="image" src="https://github.com/user-attachments/assets/01303827-33d4-45c1-9b0d-14fc044f98f0" />

This renders fine
<img width="1122" height="147" alt="image" src="https://github.com/user-attachments/assets/8811a317-efc8-4877-942f-f25159b47416" />

Reply:
I'd blame the API on this. Functions with so many parameters are hard to read.